platform/ios/src/MGLMapView.mm

```diff
@@ -230,6 +231,8 @@ @interface MGLMapView () <UIGestureRecognizerDelegate,
 @property (nonatomic) EAGLContext *context;
 @property (nonatomic) GLKView *glView;
 @property (nonatomic) UIImageView *glSnapshotView;
+@property (nonatomic, readwrite) MGLScaleBarView *scaleBarView;
```

Comment:
We expose the other views that sit on top of the map (compass, logo, attribution) as public MGLMapView properties — I think it would be consistent and useful to do the same for this scale view. At minimum, it would allow a developer to hide the scale bar.